src/code42cli/cmds/devices.py

```diff
@@ -59,8 +60,12 @@ def deactivate(state, device_guid, change_device_name, purge_date):
 
 
 def _deactivate_device(sdk, device_guid, change_device_name, purge_date):
-    device = sdk.devices.get_by_guid(device_guid)
     try:
+        int(device_guid)
```

Comment:
Could also do

```
if not str(device_guid).isnumeric():`
  raise Code42CLIError("Not a valid guid.")
```

Reply:
Why do we want to convert it into a int, the help docs in `py42` say `guid (str)`!

Reply:
We're not actually converting it, just making sure it _can_ be converted. Device guids will always be numerical, so this allows us to fail early before making an http request if we know it's gonna fail (if a column accidentally has a username in the guid column or something). 

Also, `.isnumeric()` fails if there's whitespace on either side of the guid string, where `int()` handles it fine.